examples/BasicRoutingWithResources/home.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Link, createResource, useResource } from 'react-resource-router';
+
+export const homeResource = createResource({
+  type: 'home',
+  getKey: () => 'data',
+  maxAge: 0,
+  getData: async () => {
+    let result;
+    // eslint-disable-next-line
+    try {
```

Comment:
Having this `try/catch` here is actually a bad pattern, as if you forget to re-throw you want get the error status. I suggest we remove it

Reply:
I was wondering the same. Removed it now.